admin/server/auth.py

```
@@ -104,16 +141,49 @@ def init_default_admin():
-        default_admin_rows = [u for u in users if u.email == "admin@ragflow.io"]
+        # Filter existing superuser rows by the configured
+        # ``DEFAULT_SUPERUSER_EMAIL`` *and* by active status. An
+        # inactive row matching the configured email must NOT count as
+        # the bootstrap admin — otherwise the tenant-backfill branch
+        # would skip a real active admin with a different email, and
+        # the "no matching active admin" error below would never
+        # surface.
+        default_admin_rows = [u for u in users if u.email == DEFAULT_SUPERUSER_EMAIL and u.is_active == ActiveEnum.ACTIVE.value]
         if default_admin_rows:
             default_admin = default_admin_rows[0].to_dict()
             exist, default_admin_tenant = TenantService.get_by_id(default_admin["id"])
```

Ruff (0.16.1)

[warning] Unpacked variable `default_admin_tenant` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)